src/components/LoginView.vue

```diff
@@ -78,7 +78,7 @@ export default {
 	color: var(--spotify-green);
 	text-align: center;
 	padding: 30px;
-	width: 33%;
+	width: 90%;
```

Comment:
The `90%` width should only be applied for mobile devices, desktops should remain at the `33%`. This switch should use the `only screen and (min-width: 768px)` media query. The desktop width goes in the query with the same selector, the mobile width stays in the main selector. An example of this can be seen in https://github.com/Oliver-Akins/top-lists/blob/main/src/components/cards/Track.vue